lib/uploadcare/rails/action_view/uploadcare_widget_tags.rb

```diff
@@ -0,0 +1,43 @@
+require 'uploadcare/rails/configuration'
+
+module Uploadcare::Rails::ActionView
+  module UploadcareWidgetTags
+    # A view helper to add a js script tag from CDN with just one string of code.
+    # See https://uploadcare.com/docs/uploads/file-uploader/#cdn for more info.
+    #
+    # Example:
+    #   <%= uploadcare_widget_tag %>
+    #   => <script src="https://ucarecdn.com/libs/widget/3.x/uploadcare.full.min.js"></script>
+    #      <script>
+    #        UPLOADCARE_PUBLIC_KEY = 'demopublickey';
+    #        UPLOADCARE_LOCALE = 'en';
+    #      </script>
+    #
+    # Arguments:
+    #   version: (String, default: '3.x') - version of the widget
+    #   bundle: (String, default: 'full') - https://uploadcare.com/docs/uploads/file-uploader/#bundles
+    #   min: (true/false, default: true) - sets which version to get, minified or not
+
+    UC_ORIGIN = 'ucarecdn.com'
+
+    def uploadcare_widget_tag(version: '3.x', bundle: 'full', min: true)
+      min = min == true ? '.min' : ''
+      bundle = bundle == 'default' ? '' : ".#{bundle}"
+      path = ["/libs/widget/#{version}/", "uploadcare#{bundle}#{min}.js"].join
+      uri = URI::HTTPS.build(host: UC_ORIGIN, path: path)
```

Comment:
Should we move this logic to the gem?
Is this path rails-specific?

Reply:
Somebody can use this JS in a non-Rails web application.
I think that the widget script is not Rails specific.
But this is part of a more general discussion of whether we want to make the gem so that it is as useful as possible in non-Rails web applications.

Reply:
Jfyi, as a rule of thumb we don't add anything widget specific to our basic api clients. But we add it to framework wrappers/etc.

Reply:
Got it.